src/test/java/edu/kis/powp/jobs2d/TestJobs2dPatterns.java

```diff
@@ -23,10 +25,21 @@ public class TestJobs2dPatterns {
 	 * @param application Application context.
 	 */
 	private static void setupPresetTests(Application application) {
-		SelectTestFigureOptionListener selectTestFigureOptionListener = new SelectTestFigureOptionListener(
-				DriverFeature.getDriverManager());
+		SelectTestFigureOptionListener selectTestFigureOptionListenerJoe1 = new SelectTestFigureOptionListener(
+				DriverFeature.getDriverManager() , "FiguresJoe1");
+
+		application.addTest("Figure Joe 1", selectTestFigureOptionListenerJoe1);
+
+        SelectTestFigureOptionListener selectTestFigureOptionListenerJoe2 = new SelectTestFigureOptionListener(
+                DriverFeature.getDriverManager() , "FiguresJoe2");
+
+        application.addTest("Figure Joe 2", selectTestFigureOptionListenerJoe2);
+
+		SelectTestFigureOptionListener selectTestFigureOptionListenerJane = new SelectTestFigureOptionListener(
+				DriverFeature.getDriverManager() , "FiguresJane1");
```

Comment:
You should use predefined constants:

public class SelectTestFigureOptionListener implements ActionListener {

	private DriverManager driverManager;
	private static final String FIGURE_JOE_1 = "FiguresJoe1";
	private static final String FIGURE_JOE_2 = "FiguresJoe2";
	private static final String FIGURES_JANE_1 = "FiguresJane1";